### DISCUSSION TITLE
Utilizing multiple netmaker servers?

### DISCUSSION THREAD

**Discussion by alexlyee** [gravitl/netmaker#2096] [Ideas]
I don't think this use-case is as uncommon as you may think. I wrote about mine [here](https://github.com/tonarino/innernet/issues/250), and you can see the conversation on this over at [Netbird](https://github.com/netbirdio/netbird/issues/446). I'm wondering if this is a possibility for Netmaker?

**Comment by mattkasun** [comment]
a netclient node can connect to multiple netmaker servers.

**Comment by alexlyee** [comment]
> a netclient node can connect to multiple netmaker servers.  

And it handles coordinating between them at once? I haven't seen anything on the docs on this..

**Comment by mattkasun** [comment]
https://docs.netmaker.io/netclient.html#multi-server
